base/cvd/build_external/libwebrtc/BUILD.bazel

```diff
@@ -0,0 +1,60 @@
+package(
+    default_visibility = [
+        "//visibility:public",
+    ],
+)
+
+cc_library(
+    name = "webrtc_config",
+    hdrs = [
+        "webrtc/experiments/registered_field_trials.h",
+        "webrtc/modules/audio_processing/debug.pb.h",
+    ],
+    includes = [
+      "webrtc",
+    ],
+)
+
+# These headers are needed because the libwebrtc code includes certain
+# header from these paths instead of the paths they're actually available at.
+cc_library(
+    name = "third_party",
+    srcs = [
+        "third_party/rnnoise/src/rnn_vad_weights.cc",
+    ],
```

Comment:
`base/cvd/build_external/libwebrtc/third_party/crc32/src/src` includes a lot of files that aren't mentioned in this `BUILD.bazel` file, are they being used?

Reply:
No, the @crc32c dependency should take care of that. I'll double check and delete as needed.

Reply:
Done